### ISSUE TITLE
Having issues bootsnapping discourse

### ISSUE THREAD

**Issue by SamSaffron** [Shopify/bootsnap#11]
```
bin/rails c
/Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootsnap-0.2.7/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:19:in `require': cannot load such file -- thread.rb (LoadError)
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `require'
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/2.3.0/thwait.rb:8:in `<main>'
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootsnap-0.2.7/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootsnap-0.2.7/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:240:in `load_dependency'
```

Trying 2.4.1 one now to see if it works there.

**Comment by SamSaffron** [comment]
Same error in 2.4.1

**Comment by burke** [comment]
Fixed by aec561544852f6a720886a286fb3f4978a2f6a25. I just pushed 0.2.8. Please reopen if it's still broken.

**Comment by SamSaffron** [comment]
:( still happening 

```
bin/rails c
/Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootsnap-0.2.8/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:19:in `require': cannot load such file -- thread.rb (LoadError)
	from /Users/sam/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `block in require'
```

**Comment by burke** [comment]
Ok, should be fixed for real in 0.2.10

**Comment by SamSaffron** [comment]
confirmed, this is working now! thanks heaps! checked in to discourse 

https://github.com/discourse/discourse/commit/1ae67a21f53e8d3237c7eafb0f4888bcca6f4f4a

thanks heaps!
